easybuild/easyblocks/g/gromacs.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, *args, **kwargs):
         super(EB_GROMACS, self).__init__(*args, **kwargs)
         self.lib_subdir = ''
         self.pre_env = ''
+        self.dynamic = self.toolchain.options.get('dynamic', False)
+        if LooseVersion(self.version) >= LooseVersion('4.6'):
+            self.cfg['separate_build_dir'] = True
```

Comment:
Does it fail to build when NOT using a separate build dir? If not, remove. It is the default now :)